package.json

```diff
@@ -33,24 +35,21 @@
     "gulp-replace": "^1.0.0",
     "gulp-sourcemaps": "^2.6.5",
     "gulp-tslint": "^8.1.4",
-    "gulp-typedoc": "^2.2.4",
+    "gulp-typedoc": "^2.2.5",
     "gulp-typescript": "^5.0.1",
     "gulp-uglify": "^3.0.2",
-    "lerna": "^3.18.0",
+    "lerna": "^3.22.0",
     "merge2": "^1.2.3",
     "remap-istanbul": "^0.13.0",
-    "rollup": "^1.27.14",
-    "rollup-plugin-commonjs": "^10.0.1",
-    "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-sourcemaps": "^0.4.2",
+    "rollup": "^2.12.1",
+    "rollup-plugin-sourcemaps": "^0.6.2",
     "rollup-plugin-typescript": "^1.0.1",
     "run-sequence": "^2.2.1",
     "snyk": "^1.334.0",
     "tslint": "^5.18.0",
     "tslint-microsoft-contrib": "^6.2.0",
-    "typedoc": "^0.16.9",
-    "typedoc-plugin-external-module-name": "^3.0.0",
```

Comment:
typedoc-plugin-external-module-name was removed for now.  I noticed that the module aggregation in the docs (see the right bar in docs) was broken with a typedoc release in the past which caused it to flip to files view instead of modules.  When I tried upgrading this it fixed partially but left it broken that it was actually worse as it wasn't documenting most files in the core package and elevating /Default as it is an exported namespace to be the only documented files.  I will work on this separately so I removed the unused package for now.